client/pages/advanced-search.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import parseRoute from '../lib/parse-route';
+
+const apiKey = 'key=AIzaSyAvazhS5IpqO0KVFL5XyOvDA-Gns7YyFJ8';
```

Comment:
You should not store your API Key directly in your code. Use the Webpack environment plugin to substitute in an environment variable.

https://webpack.js.org/plugins/environment-plugin/

Reply:
Also, be sure to file an issue for "secure api keys" so you don't forget later.

https://cloud.google.com/docs/authentication/api-keys#api_key_restrictions